packages/create-remix/templates/arc-stack/app.arc

```diff
@@ -0,0 +1,24 @@
+@app
+remix-aws-stack
+
+@http
+/*
+  method any
+  src server
+
+@static
+
+@aws
+region us-east-1
```

Comment:
This is totally optional, so it's up to the Remix crew, but [the `@aws` pragma](https://arc.codes/docs/en/reference/project-manifest/aws) can be omitted and Architect will default to `us-west-2`.

Reply:
I did notice that the Readme references this region setting specifically, which may be easier than mentioning Arc's default. Totally up to you :)